docs/settings.rst

```diff
@@ -21,26 +21,36 @@ More info on this at this section: :ref:`providers`.
 SCRAPY_POET_OVERRIDES
 ---------------------
 
+Deprecated. Use ``SCRAPY_POET_RULES`` instead.
+
+SCRAPY_POET_RULES
+-----------------
+
 Default: ``None``
 
 Mapping of overrides for each domain. The format of the such ``dict`` mapping
```

Comment:
Isn't it a list of AppltRule instances, not a mapping of overrides for each domain, and not a dict?

Reply:
Good catch! I missed out on this. It should actually be an `Iterable[ApplyRule]`, even from before ([reference](https://github.com/scrapinghub/scrapy-poet/pull/88/files#diff-4906cda8a37b6f984ea889eab98b820f6b9f6908f7994ab26e50adfdeff95004L92)). 

https://github.com/scrapinghub/scrapy-poet/pull/88/commits/00d5dd6224dfe247e02cdff5f706dfe201128574